lib/tests/formulae.rb

```diff
@@ -35,10 +40,115 @@ def run!(args:)
         end
 
         @skipped_or_failed_formulae_output_path.write(@skipped_or_failed_formulae.join(","))
+      ensure
+        @previously_built_bottle_cache.rmtree
       end
```

Comment:
Just remembered that this is the kind of thing you've complained about before, @Bo98. Let me know if this needs adjusting.